src/gluonts/torch/model/wavenet/estimator.py

```diff
@@ -96,7 +96,7 @@ def __init__(
         num_batches_per_epoch: int = 50,
         num_parallel_samples: int = 100,
         negative_data: bool = False,
-        trainer_kwargs: Dict[str, Any] = None,
+        trainer_kwargs: Dict[str, Any] = dict(),
```

Comment:
I think we shouldn't use mutable types as default arguments. Maybe change this to 
```python
        trainer_kwargs: Optional[Dict[str, Any]] = None,
```
and also undo the change on L181 below?

Reply:
done